lib/cisco_node_utils/encapsulation.rb

```diff
@@ -0,0 +1,100 @@
+# Copyright (c) 2014-2015 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative 'node_util'
+require_relative 'feature'
+require_relative 'vdc'
+
+module Cisco
+  # Encapsulation - node utility class for Encapsulation config mgmt.
+  class Encapsulation < NodeUtil
```

Comment:
This seems like a very generic name. Is this global encapsulation properties, per-interface encapsulation, encapsulation profile definiton, or what?